### PR TITLE
Tweak site's layout for better visual appeal

### PR DESCRIPTION
**Pull request by NinjaComics**
- Remove unwanted files and logos from the repo
- Make top-matter look short and remove projects link from navbar
- Minor changes to Module pages
- Fix floating footer by adding media queries
- Fix status-badge(tags) to render properly on smaller screens

### CODE REVIEW

**Comment by NinjaComics** [comment]
ping @n-west ?

File: pybombs-theme/templates/index.html [hunk 1]
```diff
@@ -1,44 +1,39 @@
...
         a.k.a Out Of Tree Modules that are not officially supported by
         the GNU Radio project.
-      </p>
+        </p>        
+      </div>
+      <div class="col-xs-1 col-md-1" id="headimage">
+	<a href="#front-matter-detail" class="btn btn-warning btn-fab btn-raised" 
+        data-toggle="collapse" aria-expanded="false" aria-controls="front-matter-detail" 
+        style="position: fixed; top: 290px; margin-left: 20px;">
+          <i class="fa fa-chevron-down" style="color: #e5e5e5"></i>
```
Comment (n-west) [inline pybombs-theme/templates/index.html:22]: This doesn't work with scrolling. See http://nathanwest.us/cgran-staging/

**Comment by NinjaComics** [comment]
@n-west let me know your thoughts on the latest commit !

**Comment by NinjaComics** [comment]
@n-west ping

**Comment by n-west** [comment]
Sorry that took so long. I left in the direct link to Projects anchor link and fixed the footer position. Looks great!

**Pull request closed without merging**